index.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html> 
+<html lang="en">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+    <header class="first">
+        <div class="first-img"><img src="./images/Welcome.png" alt="welcome button at the top left"></div>
+        <div class="first-img" id="hamburger" > <img src="./images/Union.png" alt="a union of three vectors"> </div>
```

Comment:
- [ ] Kindly make sure that the hamburger button has a nav tag. The nav tag is for semantics reasons. :+1: